IreneRukumbuzi/MyBrand-IreneRukumbuzi
# Ft testing

What does this PR do?
- Add Test file for testing
- Implement coveralls
What are the tasks to be completed?
- Deploying on Heroku
How can this be manually tested?
- git clone the application URL (`https://github.com/IreneRukumbuzi/MyBrand-IreneRukumbuzi.git`)
- npm run test to see the tests passing
- npm run coverage to show the percentage on coveralls
What are the relevant Trello Stories
- https://trello.com/c/JsFCAFQX/16-testing
Screenshots if appropriate
![Coveralls](https://user-images.githubusercontent.com/68101724/90763323-c6917b80-e2e6-11ea-81f5-351f382c6ee4.PNG)
